swagger/src/main/scala/org/http4s/rho/swagger/package.scala

```diff
@@ -95,6 +95,8 @@ package object swagger {
 
     def isTask: Boolean =
       t <:< typeOf[Task[_]]
-  }
 
+    def isSwaggerFile: Boolean =
+      t <:< typeOf[SwaggerFileResponse[_]]
```

Comment:
I tried that but the test doesn't pass and I don't really know why `=:=` doesn't work in that case.